stdlib/tkinter/ttk.pyi

```diff
@@ -293,8 +293,8 @@ class Combobox(Entry):
     ) -> dict[str, tuple[str, str, str, Any, Any]] | None: ...
     @overload
     def config(self, cnf: str) -> tuple[str, str, str, Any, Any]: ...
-    def current(self, newindex: Any | None = ...): ...
-    def set(self, value): ...
+    def current(self, newindex: int | None = ...) -> int: ...
```

Comment:
Note to self: passing `"end"` for `newindex` segfaults, I will report this to Tcl/Tk bug tracker